.github/workflows/docker-build.yml

```diff
@@ -3,9 +3,13 @@ name: "docker-build"
 on:
   push:
     branches:
-      - "main"
+      - "**"
```

Comment:
Instead you could just say 
```yml
on:
  push:
```
and avoid `branches`?

Reply:
I'm not sure you can based on my reading of (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags) and I'd rather not find out and reset the reviews just for the nit.